### DISCUSSION TITLE
can someone help me i have a problem in osu! lazer with incorrect display of objects and menus

### DISCUSSION THREAD

**Discussion by Bartek12282** [ppy/osu#24344] [Q&A]
![20230723_124048](https://github.com/ppy/osu/assets/140381228/64aca08e-3636-4dea-b6b3-79fc5579e0bb)
![Uploading 20230723_124056.jpg…]()
![Uploading 20230723_124105.jpg…]()
![Uploading 20230723_124114.jpg…]()

**Comment by integerrr** [comment]
Please read #15188

**Comment by Paladynee** [comment]
This is the idea & question discussions, i think this belongs to the issues page
